webextensions/manifest/optional_permissions.json

```diff
@@ -719,6 +719,46 @@
               "safari_ios": "mirror"
             }
           }
+        },
+        "webRequestFilterResponse": {
+          "__compat": {
+            "description": "<code>webRequestFilterResponse</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "110"
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "serviceWorkerScript": {
```

Comment:
Technically `serviceWorkerScript` isn't a property of a `webRequestFilterReponse` object, but a separate permission string (one is `"webRequestFilterResponse" and the other one is `"webRequestFilterResponse.serviceWorkerScript"`.

There are at least two other pre-existing WebExtensions API permissions that are using a similar convention: `"downloads"` and `"downloads.open"` and those two seems to not be nested as I'm suggesting for these two new ones:

- https://github.com/mdn/browser-compat-data/blob/8c311cba4f58c34d2c6396e65362772af23f21d5/webextensions/manifest/permissions.json#L375-L397

@rebloor given that this PR seems to have been already merged, would you mind to open a separate issue and PR to track following up about that?

Reply:
Fixed
![image](https://user-images.githubusercontent.com/7352080/219721962-bfbe76d9-e4a8-4189-a901-2daa1731643b.png)